### ISSUE TITLE
YSlow silently fails on Node >0.10

### ISSUE THREAD

**Issue by igorsantos07** [davidsonfellipe/keepfast#20]
I was running Keepfast on io.js, and YSlow was returning a strange error.
After digging the internet a bit I found it was an issue with the shell runner (?) that's not compatible with Node 0.10 (see [this issue](https://github.com/mgutz/execSync/issues/27)).

After changing to node 0.10, setting npm to use Python 2.7, and reinstalling npm packages, it started working.

As the linked issue states, it's suggested to move on to [`exec-sync`](https://github.com/mgutz/execSync/issues/27#issuecomment-85198696)

**Comment by davidsonfellipe** [comment]
@igorsantos07 Good catch! it's a important issue, I'll try a solution for app don't broke in this case.

**Comment by igorsantos07** [comment]
I've sent `phapper` a PR using native execSync: https://github.com/jmervine/phapper/pull/3
I guess once they accept that and release a new version, we will be able to reinstall dependencies and have that running smoothly :)
